.github/workflows/build.yml

```diff
@@ -771,3 +771,272 @@ jobs:
+          patchelf --force-rpath --set-rpath "$UPDATED_RPATH" libUnrealEditor-CesiumRuntime.so
+      - name: Publish plugin package artifact
+        if: ${{ success() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.BUILD_CESIUM_UNREAL_PACKAGE_NAME}}
+          path: packages
+  macOS53:
+    runs-on: macos-12
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_REGION: us-east-1
+    steps:
+      - name: Get Unreal Engine
+        run: |
+          aws s3 cp s3://cesium-unreal-engine/5.3.0/UE_53_macOS.zip .
+          unzip -q UE_53_macOS.zip -d $HOME
+          rm UE_53_macOS.zip
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0 # so that `git describe` works.
+      - name: Set environment variables
+        run: |
+          export CESIUM_UNREAL_VERSION=$(git describe)
+          export BUILD_CESIUM_UNREAL_PACKAGE_NAME="CesiumForUnreal-53-macos-${CESIUM_UNREAL_VERSION}"
+          # Make these available to subsequent steps
+          echo "CESIUM_UNREAL_VERSION=${CESIUM_UNREAL_VERSION}" >> $GITHUB_ENV
+          echo "BUILD_CESIUM_UNREAL_PACKAGE_NAME=${BUILD_CESIUM_UNREAL_PACKAGE_NAME}" >> $GITHUB_ENV
+      - name: Install nasm
+        uses: ilammy/setup-nasm@v1.4.0
+      - name: Build cesium-native
+        run: |
+          mkdir -p extern/build
+          cd extern/build
+          INSTALL_LIBDIR=`cmake .. -LA | grep CMAKE_INSTALL_LIBDIR`
+          INSTALL_LIBDIR=${INSTALL_LIBDIR##CMAKE_INSTALL_LIBDIR:PATH=}
+          cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_DEPLOYMENT_TARGET=10.15 -DCMAKE_OSX_ARCHITECTURES=arm64
+          cmake --build . -j4 --target install
+          mv $INSTALL_LIBDIR ${INSTALL_LIBDIR}-silicon
+          cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_DEPLOYMENT_TARGET=10.15 -DCMAKE_OSX_ARCHITECTURES=x86_64
+          cmake --build . -j4 --target install
+          for f in ${INSTALL_LIBDIR}/*.a
+          do
+            lipo -create -output $f ${INSTALL_LIBDIR}-silicon/$(basename -- $f) $f
+          done
+          rm -r -f ${INSTALL_LIBDIR}-silicon
+          cd ../..
+          rm -rf extern
+      - name: Build plugin
+        run: |
+          sed -i '' 's/\"EngineVersion\": \"5.0.0\"/\"EngineVersion\": \"5.3.0\"/g' CesiumForUnreal.uplugin
+          export UNREAL_ENGINE_DIR=$HOME/UE_5.3
+          cd $UNREAL_ENGINE_DIR/Engine/Build/BatchFiles
+          ./RunUAT.sh BuildPlugin -Plugin="$GITHUB_WORKSPACE/CesiumForUnreal.uplugin" -Package="$GITHUB_WORKSPACE/packages/CesiumForUnreal" -CreateSubFolder -TargetPlatforms=Mac
+      - name: Publish plugin package artifact
+        if: ${{ success() }}
+        uses: actions/upload-artifact@v3
```

Comment:
I checked the size of the macOS artifact, and it's 497MB. The other versions are ~275MB-ish.

You know of any reason for the oddly large 5.3 macOS zip?

[Run link](https://github.com/CesiumGS/cesium-unreal/actions/runs/6152360237)
`[CesiumForUnreal-53-macos-v1.30.1-18-ge6c88d7a`

Reply:
I suspect it's because the macOS distribution contains binaries for both Intel and Apple CPUs.

Reply:
Yes indeed. Here's a diff of the 52-macos artifact on left, the new 53-macos artifact on right.

The dylib binaries are about twice the size, suggesting that yes, the Apple CPU version is in there. 

Were you expecting this as output? And if so, is this something we need to note?

![image](https://github.com/CesiumGS/cesium-unreal/assets/130494071/ba28471c-2582-4160-98e6-370092b380de)

Reply:
Hmm yeah ok that's weird. UE 5.2 should have binaries for both processors, too. So I can't explain why UnrealEditor-CesiumRuntime.dylib is twice as large in 5.2 versus 5.3.

Reply:
Hmm it seems I'm wrong. I confirmed with the `lipo` tool that our 5.2 build only includes Intel binaries, no Apple ones. We do include fat binaries of the cesium-native libs, though. The build process we're using is identical between 5.2 and 5.3, so this seems to be something Epic fixed/improved? I guess I'd say this is a bug in our 5.2 build process on CI, but it's largely gone unnoticed because the binaries on the Unreal Marketplace are built by Epic, and those _do_ support both CPU architectures. The sizes of the Marketplace binaries are larger, just like our CI-built 5.3 binaries.

TL;DR: the larger binaries in our 5.3 build are a feature, not a bug.

Reply:
Ok, for the scope of this PR then (UE 5.3), things are working great.

Do we expect our 5.1 and 5.0 builds to also include Apple binaries? Because based on the sizes I'm seeing, they probably do not.

Reply:
No, I expect that 5.0 and 5.1 won't include Apple Silicon binaries. We don't even include Apple binaries for cesium-native for those versions.